gradle-plugin/src/main/groovy/io/realm/gradle/Realm.groovy

```diff
@@ -0,0 +1,17 @@
+package io.realm.gradle
+
+import com.neenbedankt.gradle.androidapt.AndroidAptPlugin
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+
+class Realm implements Plugin<Project> {
+
+    @Override
+    void apply(Project project) {
+        project.plugins.apply(AndroidAptPlugin)
```

Comment:
What happens if the user have specified this dependency himself with a different version?